concepticondata/concepticon.tsv

```diff
@@ -2115,7 +2115,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2114	BELLY OR STOMACH	The body	The lower part of the front of the torso, including the interior part below.	Person/Thing	
 2115	CEASE OR FINISH	Basic actions and technology	To stop doing something, voluntarily or involuntarily.	Action/Process	
 2116	REST OR SLEEP	Basic actions and technology	Lie down or sleep.	Action/Process	
-2117	DESIRE	Emotions and values	To have affection for someone or to want to do something.	Action/Process	
+2117	DESIRE	Emotions and values	To have affection for someone or to want to do something or to wish for something to happen.	Action/Process	
```

Comment:
All correct, but I'd keep the "have affection" at the end, as the most specific one. Thus, to want, to wish, and to have affection.